client/src/components/BusinessInfo/BusinessInfo.jsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import $ from 'jquery';
+
+class BusinessInfo extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      res_id: 1,
+      status: 'close',
+    };
+  }
+
+  componentDidMount() {
+    this.openStatus();
+  }
+
+  openStatus() {
+    const { restaurant } = this.props;
+    if (restaurant) {
+      const today = new Date();
+      const rightnow = today.getHours();
+      const amOpentime = restaurant.hours.morning.opentime;
+      const amClosetime = restaurant.hours.morning.closetime;
+      const pmOpentime = restaurant.hours.afternoon.opentime;
+      const pmClosetime = restaurant.hours.afternoon.closetime;
+      const inRange = (rightnow <= amClosetime && rightnow >= amOpentime) || (rightnow <= pmClosetime && rightnow >= pmOpentime);
+      if (inRange) {
```

Comment:
You can consider creating a variable called `status` and set it to `open` or `close` based on the hour
```suggestion
      this.setState({
          status: status,
        });
```